### ISSUE TITLE
Is there Linux support yet?

### ISSUE THREAD

**Issue by Eskpil** [alaingalvan/CrossWindow#7]

**Comment by alaingalvan** [comment]
Not at the moment, there will be, I just haven't had the time to focus on that platform, I've been focused on Win32/MacOS/iOS for now.

There's a lot of folks that would want a library like this for Linux though, this might be the push I need to keep working on it! 💪

**Comment by Eskpil** [comment]
Yeah I am there where it would be really great with Linux support, I am looking to make a cross-platform desktop window library in nodejs which is cross-platform and works differently from libraries like electron.
